taxi/tests/test_search.py

```diff
@@ -0,0 +1,87 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+
+class SearchDriver(TestCase):
+    def setUp(self) -> None:
+        self.user = get_user_model().objects.create_user(
+            "test",
+            "pass123",
+        )
+        self.client.force_login(self.user)
+
+    def test_search_driver(self):
+        form = {
+            "username": "test",
+        }
+        get_user_model().objects.create(
+            username="driver", license_number="ASD41244"
+        )
+        filtered_drivers = get_user_model().objects.filter(
+            username__icontains=form["username"]
+        )
+        response = self.client.get(
+            reverse("taxi:driver-list") + "?username=" + form["username"]
```

Comment:
No need to fix it right now, but for future: 
![image](https://user-images.githubusercontent.com/44442087/216758048-7620b80d-38fc-4811-9e52-7c74b2db09ee.png)

This way you can pass query parameters without changing the url